tests/services/test_storage_routing_service.py

```
@@ -26,26 +26,26 @@ def test_select_storage_location_multiple_scoring(self, db_session, monitored_pa
-        
+
         loc2_path = tmp_path / "loc2"
         loc2_path.mkdir()
         loc2 = ColdStorageLocation(name="Loc 2", path=str(loc2_path))
-        
+
         db_session.add_all([loc1, loc2])
         db_session.commit()
-        
+
         path = monitored_path_factory("Multi Route", "/tmp/hot_multi")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/hot_multi"

(S108)